src/common/utils.hpp

```diff
@@ -74,13 +75,19 @@ static_assert(sizeof(void *) == 8, "oneDNN supports 64-bit architectures only");
 #define CHECK(f) \
     do { \
         dnnl::impl::status_t _status_ = f; \
-        if (_status_ != dnnl::impl::status::success) return _status_; \
+        if (_status_ != dnnl::impl::status::success) { \
+            VDEBUGINFO(debug, common, utils, "CHECK() failed"); \
```

Comment:
Maybe
```suggestion
            VDEBUGINFO(debug, common, utils, #f " = %s", dnnl_status2str(f)); \
```
is a more useful message for debugging.